examples/ego_ped_example.py

```diff
@@ -56,7 +56,7 @@ def get_file():
 
 def main():
 
-    map_def = convert_map("maps/svg_maps/debug_03.svg")
+    map_def = convert_map("maps/svg_maps/debug_06.svg")
 
```

Comment:
Why did you change the svg map? Pure curiosity.

Reply:
I deleted debug_03, because it was a map I build a the start and it didn't provide any specific value.